src/lib/Sympa/Request/Handler/subscribe.pm

```diff
@@ -118,9 +118,7 @@ sub _twist {
     $user->lang($list->{'admin'}{'lang'}) unless $user->lang;
     $user->save;
 
-    ## Now send the welcome file to the user
-    $request->{quiet} = ($Conf::Conf{'quiet_subscription'} eq "on")
-        if $Conf::Conf{'quiet_subscription'} ne "optional";
```

Comment:
I think we should keep that: deporting the `quiet` parameter in the templates, even if it’s hidden, allows users to bypass the `quiet_subscription` global parameter by editing the HTML code of the page.

Same thing in `src/lib/Sympa/Request/Handler/add.pm`

Reply:
See the commit c732ce9 .  When `quiet_subscription` is `optional`, it needs to be handled in the template.  Thus, when it is `on`, it may also be handled in the template. In the end, I decided that the lines in Perl sources were not necessary.

Reply:
Correction: See commit 93dc8b7 .

Reply:
I understand that some things need to be handled in templates, but settings that should not be overriden by users should always be enforced in the backend otherwise users may bypass those settings.

Reply:
I think that removing the check in the backend will also allow to bypass the setting when using mail commands 🤔